<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>5GTS Controller</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>指令发送与返回</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>RMTP服务器配置</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>IP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtRmtpIP">
               <property name="text">
                <string>172.25.65.122</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Port</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtRmtpPort">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>5000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRmtpListen">
               <property name="text">
                <string>开始监听</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRmtpStop">
               <property name="text">
                <string>停止监听</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>5GTS 设备配置</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="horizontalSpacing">
              <number>6</number>
             </property>
             <item row="0" column="5">
              <widget class="QLineEdit" name="txtDataPort">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>5565</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="txtDevicePort">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>5563</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="txtDeviceIp">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string>172.25.69.157</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>IP</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>控制端口</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>数据端口</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QPushButton" name="btnLinkDevice">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>链接设备</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2" colspan="2">
              <widget class="QPushButton" name="btnSendCmd">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>开始测向</string>
               </property>
              </widget>
             </item>
             <item row="4" column="4" colspan="2">
              <widget class="QPushButton" name="btnStop">
               <property name="text">
                <string>停止测向</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>开始服务</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>停止服务</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnOptimize">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>数据校准</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="txtDeviceCmd">
            <property name="plainText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="txtDeviceResponse"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>指令接收与返回</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTextEdit" name="txtRmtpCmd"/>
          </item>
          <item>
           <widget class="QTextEdit" name="txtRmtpResponse"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtDeviceIp</tabstop>
  <tabstop>txtRmtpResponse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
